packages/react/src/components/dropdown/combobox/Combobox.test.tsx

```diff
@@ -61,7 +61,7 @@ describe('<Combobox />', () => {
       // Choose one option
       fireEvent.click(visibleOptions[0]);
       // Ensure that it's visible in selected items
-      expect(queryAllByText(options[0].label).length).toEqual(1);
+      expect(queryAllByText(options[0].label).length).toEqual(2);
```

Comment:
This test broke as I updated the jsdom from jsdom-fourteen -> jest-environment-jsdom-sixteen. I don't know how it has worked before because the text should be twice in the component (selected item tag and select list)?

Reply:
I tested it manually and it seemed to stay visible in the select list and in the tag list 🤔

Reply:
Yes, the test renders components like this: 
```html
<span aria-hidden="true">Finland</span>
```
```html
<li role="option" aria-selected="true" id="hds-combobox-1-item-0" class="menuItem selected"><span class="checkbox" aria-hidden="true"><svg class="icon s" viewBox="0 0 24 24" role="img" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><rect width="24" height="24"></rect><polygon fill="currentColor" points="21 7 10 18 4.5 12.5 6 11 10 15 19.5 5.5"></polygon></g></svg></span>Finland</li
```